source/_components/sensor.filter.markdown

```diff
@@ -0,0 +1,102 @@
+---
+layout: page
+title: "Filter"
+description: "Instructions how to integrate Filter Sensors into Home Assistant."
+date: 2018-02-20
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Sensor
+ha_release: 0.12
+ha_iot_class: "Local Push"
+logo: home-assistant.png
+---
+
+The `filter` component supports sensors which process the **numeric states** from other entities. `filter` will apply a signal processing algorithm to a serie of states and produce a `new state` given the choosen algorithm.
+
+To enable Filter Sensors in your installation, add the following to your `configuration.yaml` file:
+
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: filter
+    entity_id: sensor.relative_humidity
+    name: filtered relative humidity 
+    filter: lowpass
+    options:
+      time_constant: 10
+  - platform: filter
+    entity_id: sensor.temperature
+    name: filtered temperature
+    filter: outlier
+    options:
+      constant: 0.10
```

Comment:
is this time_constant? it's not listed again in the details below

Reply:
no... it's **radius**... will correct